tensorrt_llm/_torch/auto_deploy/custom_ops/attention_interface.py

```
@@ -572,6 +572,16 @@ def _store_arg(
             # pin the memory on the host
             tnsr_host = torch.tensor(tnsr_like, dtype=tnsr_device.dtype, pin_memory=True)
 
+            # check if we need/allow a resize
+            if tnsr_device.numel() < tnsr_host.numel():
+                if reset:
+                    tnsr_device.resize_(tnsr_host.numel())
+                else:
+                    raise ValueError(
+                        f"tensor {name} is too small, available: {tnsr_device.numel()}, "
+                        f"required: {tnsr_host.numel()}"
+                    )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)